cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -216,9 +225,9 @@ if(PARQUET_BUILD_TESTS)
 
   if("$ENV{GTEST_HOME}" STREQUAL "")
     if(APPLE)
-      set(GTEST_CMAKE_CXX_FLAGS "-fPIC -std=c++11 -stdlib=libc++ -DGTEST_USE_OWN_TR1_TUPLE=1 -Wno-unused-value -Wno-ignored-attributes")
```

Comment:
That's because all the options I removed are already included in parquet-cpp's `CMAKE_CXX_FLAGS`.